analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt

```diff
@@ -36,21 +41,31 @@ import kotlinx.coroutines.launch
  * Main screen layout for the sample app
  */
 @Composable
-fun MainScreen(sampleName: String, application: Application) {
+fun MainScreen(sampleName: String) {
     // coroutineScope that will be cancelled when this call leaves the composition
     val sampleCoroutineScope = rememberCoroutineScope()
+    // get the application property that will be used to construct MapViewModel
+    val sampleApplication = LocalContext.current.applicationContext as Application
     // create a ViewModel to handle MapView interactions
-    val mapViewModel = remember { MapViewModel(application, sampleCoroutineScope) }
+    val mapViewModel = remember { MapViewModel(sampleApplication, sampleCoroutineScope) }
+    // create a Viewpoint
+    val viewpoint = Viewpoint(
+        center = Point(-13671170.0, 5693633.0, SpatialReference(wkid = 3857)),
+        scale = 1e5
+    )
 
     Scaffold(
         topBar = { SampleTopAppBar(title = sampleName) },
         content = {
             // sample app content layout
             Column(modifier = Modifier.fillMaxSize().padding(it)) {
-                // composable function that wraps the MapView
-                ComposeMapView(
-                    modifier = Modifier.fillMaxSize().weight(1f),
-                    mapViewModel = mapViewModel
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISMap = mapViewModel.map,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = viewpoint),
+                    onSingleTapConfirmed = {}
```

Comment:
Previously, we have a default value defined for ComposableMapView as fun ComposeMapView(/*...*/, onSingleTap: (SingleTapConfirmedEvent) -> Unit = {}), but no value is set for onSingleTap when calling ComposeMapView. So it actually does nothing.

After migrating to use composable MapView, should I intentionally keep it here as MapView(/*..*/, onSingleTapConfirmed = {}), or should I remove this?